script.js

```diff
@@ -0,0 +1,228 @@
+var audioError = function(){
+  return new Howl({
+    urls: ['/home/amanda/Brackets/javaScript/simonGame/beep2.mp3'],
+    volume: 0.5,
+  })
+}
+
+var audioGreen = function(callback){
```

Comment:
Seems that you are using the same code on audioGreen, audioRed and so on. 

In these cases you can just create one function and pass the values that are differents as variables, so it would be something like:
```
+var audio = function(urls, id, color, callback){
 +  return  new Howl({
 +    urls: urls,
 +    volume: 0.5,
 +    onend: function() {
 +      $(id).css('background',color);
 +      if (callback){
 +        callback();
 +      }
 +    }
 +  }); 
 +}
```